### PR TITLE
SE-2092 First commit of TTPR notebook

### PR DESCRIPTION
**Pull request by DavidWassellFinbourne**

### CODE REVIEW

File: examples/use-cases/post-trade-processing/Trade To Portfolio Rate.ipynb [hunk 1]
```diff
@@ -0,0 +1,569 @@
+{
```
Comment (slemasne) [inline examples/use-cases/post-trade-processing/Trade To Portfolio Rate.ipynb:1]: <p>* Load quotes and recipe first</p><p>* Load transactions after</p><p>* Note - you have both sections as "4"</p><p>* Also, add more details under the "Load Transaction" about what is happening</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/435/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (slemasne) [inline examples/use-cases/post-trade-processing/Trade To Portfolio Rate.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #35.            "properties.Transaction/default/TradeToPortfolioRate.value.metric_value.value" : "TTPR"
</pre><p>Nice! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/435/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

[the remark this answers is not in the document]
Comment (rizwansaeed) [reply examples/use-cases/post-trade-processing/Trade To Portfolio Rate.ipynb:1]: <p>also add the meta data doc strings so it can be indexed</p>

Comment (rizwansaeed) [inline examples/use-cases/post-trade-processing/Trade To Portfolio Rate.ipynb:1]: <p>this doesn't feel like it should be in the notebook, what are we actually trying to show?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/435/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (rizwansaeed) [inline examples/use-cases/post-trade-processing/Trade To Portfolio Rate.ipynb:1]: <p>this needs more commentary explaining what the use is seeing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/435/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (DavidWassellFinbourne) [reply examples/use-cases/post-trade-processing/Trade To Portfolio Rate.ipynb:1]: <p>Without renaming, the table is not pleasant on the eyes</p>

**Comment by DavidWassellFinbourne** [comment]
Outdated PR

**Pull request closed without merging**